articles/support/tickets.md

```diff
@@ -19,23 +19,40 @@ If you are an existing PSaaS Appliance customer, you will need to create an Auth
 ![Support Center](/media/articles/support/open-ticket.png)
-1. Choose the **Affected Account** from the dropdown menu.
-1.  Under **What can we help you with?** select the type of issue that best fits your case.
+1. Choose the **Affected Tenant** from the dropdown menu.
+1. Under **Issue Type** select the type of issue that best fits your case.
 ![Issue types](/media/articles/support/issue-types.png)
-    * **Auth0 Service Issue** - Choose this if something isn’t working correctly or you need help using a feature or getting something working.
-        * With Service Issues, you will be asked to select a severity:
+    * **Public Cloud Support Incident** or **Appliance Support Incident** - (the second option is only available to Appliance customers)
+        * If you are not a Trial Tennant, you will be asked to select a severity:
             *  **Low: Product Question** - You have questions about how things work, whether we support something
             * **Normal: General Support Question** - You are in development and have a question about how to configure something or how to resolve an error
             * **High: Production Application Issue** - Your Production application is experiencing a minor outage impacting some users/feature(s)
             * **Urgent: Production Application Offline** - Your Production application is experiencing a critical outage impacting all users
-        *  You will then be asked to choose an option under "What is the service issue related to?"
-            *  _You may be prompted for additional information depending on your selection._
+        *  Depending on the issue type you chose after you select the severity, you may need to answer the **What can we help you with?**. Choose one of the following:
+            * **Something is broken**
+            * **How to** 
+            * **Enhancement/Feature Request**
+            * **Billing or Payment**
+            * **Compliance/Legal**
+            * **Outage**
+            * **Do not understand self-service resources**
+            * **Enterprise Connections**
+            * **SDK/Library**
+            * **Management API**
+            * **Custom DB Connection**
+            * **SSO Integrations**
+![Customer Reason](/media/articles/support/customer-reason.png)
+        * After you make your selection, you can choose to answer some follow-up questions to further refine your request. You can also choose to skip this step by clicking the toggle. 
+![Follow-up questions](/media/articles/support/follow-up.png)
     * **Billing, Payments, or Account Issues** - You are experiencing issues such as incorrect billing, charges to the incorrect account, and so on
     * **Auth0 Service Feedback** - You would like to suggested product improvements or enhancement requests
     * **Vulnerability Reports or Legal Issues** - You would like to report security vulnerabilities or legal questions
```

Comment:
Compliance/Legal

Reply:
Fixed.

Reply:
Question: Are the rest of the options still available: Outage, Do not understand self-service resoures, Enterprise Connections, SDK/Library, Management API, Customer DB Connection, SSO Integrations?